Autocoders/Python/src/fprime_ac/generators/templates/gtest/cpp.tmpl

```diff
@@ -157,7 +157,7 @@ $emit_cpp_params([ $param_fileName, $param_lineNumber, $param_index, $param_val
       $failure_message
       $LTLT "  Value:    Value at index "
       $LTLT __index
-      $LTLT " on telmetry channel ${channel_name}\n"
+      $LTLT " on telemetry channel ${channel_name}\n"
```

Comment:
This changes something...

Reply:
This should be ok.  Changes a message in the output of a test failure.